docs/python-api.rst

```diff
@@ -321,9 +321,9 @@ Binary columns
 Columns storing binary data take the same approach as
 :ref:`sec_tables_api_text_columns` to encoding
 :ref:`variable length data <sec_encoding_ragged_columns>`.
-The difference between the two is
-only raw :class:`bytes` values are accepted: no character encoding or
-decoding is done on the data. Consider the following example::
+Consider the following example where a table has no
+``metadata_schema`` such that arbitrary bytes can be stored. (This is not recommended
```

Comment:
Maybe "such that arbitrary bytes can be stored and no automatic encoding or decoding of objects is performed 
(see :ref:`sec_metadata_XXX` for details)"